{{cookiecutter.project_name}}/noxfile.py

```diff
@@ -45,48 +40,37 @@ def tests(session: nox.Session) -> None:
     """
     Run the unit and regular tests.
     """
-    session.install(".[test]")
+    session.install("-e.[test]")
```

Comment:
```suggestion
    session.install(".[test]")
```

Still running into an issue with this on UV.

Reply:
Actually, I think it's a meson-python issue with not using ninja from the system in the editable mode.